documentation/cobigen-intellij-installation.asciidoc

```diff
@@ -10,40 +10,54 @@ toc::[]
 If you want to use IntelliJ for CobiGen project, you can follow the steps below:
 
 . You can easily install IntelliJ, create a launch script for it etc. as described in
-
++
 `https://github.com/devonfw/ide/blob/master/documentation/intellij.asciidoc`
 
 . After starting IntelliJ, you **need** to import the cobigen folder as maven project
 
 . Next you **have** to configure the maven settings in IntelliJ as follows:
 
 .. Go to `File`-> `Settings` -> `Build, Execution, Deployment` -> `Build Tools` -> `Maven`
-
-... **Choose** `(Maven home Path)` from your devonFolder -> devonProjects/software/Maven
++
+--
+... **Choose** `(Maven home Path)` from your devonFolder -> devonProjects/software/maven
 
 ... **Override** `(user settings file)` -> devonProjects/conf/.m2/settings.xml
 
 ... **Override** `(Local repository)` -> devonProjects/conf/.m2/repository
-
-
+--
++
 image::images/howtos/intellij/maven-settings.png[Maven Settings]
 
-.. Now go to `File` -> `Project structure` -> `Project Settings` -> `Project add java sdk version from` -> `devonProjects/software/java`
-
+.. Now go to `File` -> `Project Structure` -> `Project Settings` -> `Project add java sdk version from` -> `devonProjects/software/java`. 
++
+Make sure you are using Java 11. If you have set up your devonfw-ide with this settings URL `https://github.com/devonfw/cobigen-ide-settings.git`, the correct version will be set automatically. If not, you can download Java 11 manually and place it into `devonProjects/software/java11`.
```

Comment:
Can be removed or adjusted to >Java11
Otherwise the java11 folder needs to be selected in intellij afterwards.

Reply:
I am not sure which Java version is required, maybe you or someone else could clarify that again. 
I thought you mentioned earlier that it should be Java 11, but it seems that I misunderstood.

Reply:
For now we can keep Java11 I think. Although the new setup script installs a Java17 version into software/extras. I don't know how to handle this yet.